app/Notifications/SlackTagNotification.php

```diff
@@ -5,21 +5,12 @@
 use Rogue\Models\Tag;
 use Rogue\Models\Post;
 use Rogue\Services\GraphQL;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Str;
 use Illuminate\Bus\Queueable;
 use Illuminate\Notifications\Notification;
 use Illuminate\Notifications\Messages\SlackMessage;
 
-const SLACK_NOTIFICATION_QUERY = '
-    query SlackNotificationQuery($userId: String!, $adminId: String!) {
-        user(id: $userId) {
-            displayName
-        }
-        admin: user(id: $adminId) {
-            displayName
-        }
-    }
-';
-
```

Comment:
Removed this query from here and added a new method to the `GraphQL` class, so that we can easily [mock the request](https://github.com/DoSomething/rogue/pull/1071/files#diff-506a2b9afeee7eb30e7ce345deedc128R69-R76).

Reply:
Ah, got it.

I prefer keeping queries colocated with their usage if we can (since it's easier to reason about what data is being fetched, and how changing a query would affect anything that depends on it). I wonder if there's a way we could keep this query here and instead mock the request "further down" based on the `SlackNotificationQuery` name (like we do in Cypress tests!)

Reply:
Not something we need to sort out now, but I'd love to consider thinking about this...

Reply:
🤔 that's fair! And your [suggestion](https://github.com/DoSomething/rogue/pull/1071#discussion_r462356992) seems like it would resolve the testing issues I was having.

Reply:
Restored old approach since the PHPUnit env var setting keeps notifications from triggering and solves the need to mock this.